### ISSUE TITLE
PythonGUIInputOutput is missing error in sample and demo code for Delphi Community Edition 10.3 and Lazarus FPC 2.0.10

### ISSUE THREAD

**Issue by csevast** [pyscripter/python4delphi#233]
After your fix the package is installing correctly but the demo and sample code is not running because of the missing file mentioned in the subject. I checked the similar issues of the last days and I figured out that the issue is connected with the python4delphi migration to the new 10.4 version of Delphi.

**Comment by pyscripter** [comment]
The unit is now called Lcl.PythonGUIInputOutput and located in the Lcl directory.  The demo will be fixed shortly.

**Comment by csevast** [comment]
Nice, I changed the calling name of the unit and all four demos for Lazarus can be compiled now. But when I tried them pressing their form buttons —I didn't check what their purpose was— in some of them error messages rise. I will look at them in more detail another time. 

For the Delphi projects I added in the Delphi Library Path the `.\Source` and the `.\Source\vlc` and everything is looking to to work fine.

Thank you for your fast response.
